app.py

```diff
@@ -14,6 +15,7 @@
 
 class Translate(Resource):
     def get(self):
+        bootstrap()
```

Comment:
Although I was the one to suggest this, I now am thinking that maybe we should add a constructor so that bootstrap is called whenever Translate class is invoked - even if get request isn't. I just added a TODO in my current branch so do not change this now as it will create later merge issues, but wanted to just document this somewhere

Reply:
Sounds good, and I agree that seems like a better idea

Reply:
Yup! I actually just fixed it on my current language detection branch so it will be resolved shortly!

Reply:
+1 to this